### PR TITLE
#2725 and #2730

### PR DESCRIPTION
**Pull request by Dieho**

### CODE REVIEW

**Comment by scrawl** [comment]
Please configure your editor to use 4 spaces, not tabs.

File: apps/opencs/view/world/dialoguesubview.hpp [hunk 1]
```diff
@@ -237,11 +237,13 @@ namespace CSVWorld
 
             bool isLocked() const;
 
+			int GetColumnIndex(CSMWorld::Columns::ColumnId column);
```
Comment (scrawl) [inline apps/opencs/view/world/dialoguesubview.hpp]: Naming should be ```getColumnIndex``` to be consistent with the existing methods in this file.

**Comment by zinnschlag** [comment]
Also, don't add a getColumnIndex function to the Dialogue class. That is redundant and very inconsistent with what we are doing everywhere else.

**Comment by smbas** [comment]
The 0 column in tables is the ID column (i.e. ColumnId_Id).

File: apps/opencs/view/world/dialoguesubview.hpp [hunk 2]
```diff
@@ -236,12 +236,12 @@ namespace CSVWorld
...
 
             bool isLocked() const;
-
+        
         public:
 
             SimpleDialogueSubView (const CSMWorld::UniversalId& id, CSMDoc::Document& document);
 
-            virtual void setEditLock (bool locked);
+	    virtual void setEditLock (bool locked);
```
Comment (scrawl) [inline apps/opencs/view/world/dialoguesubview.hpp]: There's still tabs here.

**Pull request merged**